### PR TITLE
Condensing hsl and hsv conversions

### PR DESCRIPTION
**Pull request by toish**
This PR removes the HSV=>RGB conversion, because it's pretty much a duplicate of HSL=>RGB. Also using a desaturated version of test colour for shade as well. The one we were using had a 'v' value of 100, so it couldn't express a brighter version.

### CODE REVIEW

**Comment by TehShrike** [comment]
aw, no chance for peer review?

**Comment by toish** [comment]
Oops, sorry! Was thinking because it merged into refactor I'd just merge it in, bad choice on my part! If you don't mind looking it over quick, will revert it if you find anything!

File: test/test.js [hunk 1]
```diff
@@ -189,10 +189,10 @@ describe('Operations', function () {
   })
 
   it('should match expected value for shade', () => {
-    assert(closeXYZ(chroma.shade(20, '#4fc7ff').XYZ,
-      { X: 41.69787317690973,
-        Y: 49.72911239090596,
-        Z: 102.00870969130528 }
+    assert(closeXYZ(chroma.shade(20, '#4fc700').XYZ,
+      { X: 39.303512967548,
```
Comment (TehShrike) [inline test/test.js]: What's the reasoning for the different input/output?

Comment (toish) [inline test/test.js]: #4fc7ff was a fully saturated colour, so the v value was "100", increasing the shade by 20 would return the same value, so I toned back the initial colour's shade

**Pull request merged**